content/applications/hr/time_off/my_time.rst

```diff
@@ -0,0 +1,106 @@
+
+To reset the view so it includes the current date, click the :guilabel:`Today` button.
+
+Above the calendar view is a summary of the user's time off balances. Every time off type that has
+been allocated appears in its own summary box. Each summary lists the type of time off, the
+corresponding icon, the current available balance (in hours or days), and an expiration date (if
+applicable).
+
+To view the full details of a time off balance, click the :icon:`fa-question-circle-o`
+:guilabel:`(question mark)` icon at the end of the :guilabel:`(DAYS/HOURS) AVAILABLE` on the time
+off summary. The complete details are presented in a popover window, including the
+:guilabel:`Allocated` time, future :guilabel:`Accrual` time, :guilabel:`Approved` time off
+scheduled, :guilabel:`Planned` time off, and the currently :guilabel:`Available` time off.
+
+.. image:: my_time/balance-details.png
+   :align: center
+   :alt: A view of the complete time off balance details in the popover window.
+
+A user can also select a future date to see an estimate of how much time they should accrue by that
+ point. On the right side of the time off summary blocks, there is a :guilabel:`Balance at the
+ (date)` field. Click on the date, and a calendar selector popover appears.
+
+The current date is the default date selected. Navigate to the desired date, and Odoo displays the
+time off balances the user should have on that date. This takes into account all time off currently
+planned and approved. To return to the current date, click the :guilabel:`Today` button to the right
+of the date field.
+
+On the right side of the calendar, the various time off types are displayed, with their
+corresponding colors. The :guilabel:`Legend` explains how the various statuses for time off requests
+are presented.
+
+Time off that has been validated appears in a solid color. Time off requests that still need
+to be approved appear with white stripes in the color. Refused time off requests have a line
+through the dates. The color for each request corresponds to the specified color set in the
+:guilabel:`Time Off Type` section above the :guilabel:`Legend`).
+
+New time off requests can be made from the :guilabel:`Dashboard`. Click the :guilabel:`New` button
+at the top of the dashboard, and a :doc:`New Time Off <request_time_off>` pop-up window appears.
+
+New allocation requests can also be made from the :guilabel:`Dashboard`. Click the :guilabel:`New
+Allocation Request` button at the top of the :guilabel:`Dashboard` to request more time off, and a
+:ref:`New Allocation <time_off/request-allocation>` modal appears.
+
+.. image:: my_time/dashboard.png
+   :align: center
+   :alt: Time off dashboard view with the legend, time off summaries, and view buttons highlighted.
+
+.. _time_off/my-time-off:
+
+My time off
+-----------
+
+To view a list of all the user's time off requests, navigate to :menuselection:`Time Off app --> My
```

Comment:
```suggestion
To view a list of all the time off requests, navigate to :menuselection:`Time Off app --> My
```

Reply:
The reason I kept saying "user's" throughout the doc was because if you are a manager, you see both your requests and the people you manage, in some places. other spots, just yours. So I wanted to make that more clear, but I see why it wounds/reads better without it.